std/node/util.ts

```diff
@@ -4,6 +4,25 @@ import * as types from "./_util/_util_types.ts";
 
 export { types };
 
+const DEFAULT_INSPECT_OPTIONS = {
+  showHidden: false,
+  depth: 2,
+  colors: false,
+  customInspect: true,
+  showProxy: false,
+  maxArrayLength: 100,
+  maxStringLength: Infinity,
+  breakLength: 80,
+  compact: 3,
+  sorted: false,
+  getters: false,
+};
```

Comment:
Can you consume these defults in the `inspect()` function below?
```diff
-   return Deno.inspect(object, {
-    depth: opts.depth ?? 4,
-    iterableLimit: opts.iterableLimit ?? 100,
-    compact: !!(opts.compact ?? true),
-    sorted: !!(opts.sorted ?? false),
-   });
+   opts = { ...opts, ...DEFAULT_INSPECT_OPTIONS};
+   return Deno.inspect(object, {
+    depth: opts.depth,
+    iterableLimit: opts.maxArrayLength,
+    compact: !!opts.compact,
+    sorted: !!opts.sorted,
+   });
```

Reply:
Thanks for the suggestion!